src/Channel.h

```diff
@@ -71,7 +71,23 @@ class Channel : public QObject {
 
 #ifdef MUMBLE
 	unsigned int uiPermissions;
-	bool bFiltered;
+
+	/// Visibility modifiers used when applying the channel filter
+	/// The channel the user is in will be always visible
+	/// This enum is used in the client DB, only append new entries
+	enum FilterMode {
+		/// Channel is filtered, if it is empty and 'Automatically hide empty channels' is active
+		FILTER_MODE_NORMAL,
+		/// Channel is filtered, if the channel filter is active
+		FILTER_MODE_HIDE,
+		/// Channel is not filtered, even if it is empty
+		FILTER_MODE_PIN
+	};
```

Comment:
Please define the new enum in a separate header file and make it an enum _class_ rather than a plain C-style enum.